internal/metrics/metrics.go

```diff
@@ -88,3 +105,7 @@ func RegisterStatuserMetrics() {
 func RegisterApiMetrics() {
 	prometheus.MustRegister(JobQueueSize)
 }
+
+func RegisterWorkerMetrics() {
+	prometheus.MustRegister(JobQueueSize)
```

Comment:
it is supposed to be `BackgroundJobDuration` isnt it?

Reply:
Damn good eyes! I tested only that the metrics appear in general, but it actually DOES NOT WORK without this.

Reply:
What about registering `JobQueueInFlight`?